sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/Clauses/FromClause.cs

```diff
@@ -8,21 +8,19 @@ namespace Azure.DigitalTwins.Core.QueryBuilder
     /// <summary>
     /// Custom object for a FROM clause. Only meant to be used when adding FROM to a query. Hidden from user.
```

Comment:
Why `Hidden from user.`? Because the class is internal? You wouldn't need to add that comment to every class (and property, and method) that is not public.